apps/server/src/shared/domain/domainobject/board/types/board-composite-visitor.ts

```diff
@@ -0,0 +1,18 @@
+import type { Card } from '../card.do';
+import type { ColumnBoard } from '../column-board.do';
+import type { Column } from '../column.do';
+import type { TextElement } from '../text-element.do';
+
+export interface BoardCompositeVisitor {
+	visitColumnBoard(columnBoard: ColumnBoard): void;
+	visitColumn(column: Column): void;
+	visitCard(card: Card): void;
+	visitTextElement(textElement: TextElement): void;
```

Comment:
Same comment like to the super class. I think we should be avoid also super interfaces and i think the real visistor is each of the method at the moment and not the BoardCompositeVisitor.